src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/service/cessations/BeneficialOwnerCessationService.java

```diff
@@ -0,0 +1,111 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.service.cessations;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.tuple.Pair;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.beneficialowner.PrivateBoDataApi;
+import uk.gov.companieshouse.api.model.psc.PscApi;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.BeneficialOwnerCorporateDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.BeneficialOwnerGovernmentOrPublicAuthorityDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.BeneficialOwnerIndividualDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.cessations.*;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.types.PersonName;
+
+@Service
+public class BeneficialOwnerCessationService {
+
+  public static final String NO_PAIR_FOUND = "No pair found for the provided hashedId";
+
+  public List<Cessation> beneficialOwnerCessations(
+          OverseasEntitySubmissionDto overseasEntitySubmissionDto,
+          Map<String, Pair<PscApi, PrivateBoDataApi>> combinedBoData) {
+
+    List<Cessation> cessations = new ArrayList<>();
+    cessations.addAll(returnIndividualBeneficialOwners(overseasEntitySubmissionDto, combinedBoData));
+    cessations.addAll(returnCorporateEntityBeneficialOwnerCessations(overseasEntitySubmissionDto, combinedBoData));
+    cessations.addAll(returnLegalPersonBeneficialOwners(overseasEntitySubmissionDto, combinedBoData));
+    return cessations;
+  }
+
+  private List<IndividualBeneficialOwnerCessation> returnIndividualBeneficialOwners(
```

Comment:
Rename to:
```suggestion
  private List<IndividualBeneficialOwnerCessation> returnIndividualBeneficialOwnerCessations(
```

Reply:
make it get instead of return as well

Reply:
Thought I had changed all of these. Done.